data/sql/updates/pending_db_world/rev_1610739863539733975.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1610739863539733975');
+
+-- "Chest of The Seven" from Blackrock Depths should spawn in a different position
+UPDATE gameobject SET position_x="1265.55", position_y="-284.421", position_z="-78.2193", orientation="0.786698", rotation2="-0.383284", rotation3="-0.923631" WHERE guid=67872;
```

Comment:
add backticks ( \` ) around keywords and remove the double quotes around all numbers ( " )

Reply:
Done!